source/frontend/src/features/acquisition/hooks/useAcquisitionFileExport.ts

```diff
@@ -0,0 +1,50 @@
+import axios, { AxiosError } from 'axios';
+import fileDownload from 'js-file-download';
+import { useCallback } from 'react';
+import { useDispatch } from 'react-redux';
+import { hideLoading, showLoading } from 'react-redux-loading-bar';
+import { toast } from 'react-toastify';
+
+import {
+  IPaginateAcquisition,
+  useApiAcquisitionFile,
+} from '@/hooks/pims-api/useApiAcquisitionFile';
+import { IApiError } from '@/interfaces/IApiError';
+import { logRequest, logSuccess } from '@/store/slices/network/networkSlice';
+
+export const useAcquisitionFileExport = () => {
+  const { exportAcquisitionFiles: apiExportAcquisitionFiles } = useApiAcquisitionFile();
+  const dispatch = useDispatch();
+
+  const exportAcquisitionFiles = useCallback(
+    async (
+      filter: IPaginateAcquisition,
+      outputFormat: 'csv' | 'excel' = 'excel',
```

Comment:
I don't think that csv is supported here so can probably simplify this.

Reply:
simplified,